app/controllers/documents/ids_controller.rb

```diff
@@ -3,6 +3,7 @@ class IdsController < DocumentUploadQuestionController
     def edit
       @title = "Attach a photo of your ID card"
       @help_text = "The IRS requires us to see a current drivers license, passport, or state ID."
+      # TODO: replace this with name from intake
```

Comment:
this is one of a few places where we get the client's name from the user record. is it part of this story to replace these with the new name column on intake?